.gitignore

```diff
@@ -2,3 +2,5 @@ dist/
 node_modules/
 types/
 .vscode
+!.env.template
```

Comment:
Since you've renamed the file from `.env.template` to `.env.example` this will need to be updated in `.gitignore` as well

Reply:
resolved in 7d2f512e50ac52cb4a1c6de1f8d7f674a6a95f41